addons/hr_recruitment_survey/models/survey_invite.py

```diff
@@ -9,14 +9,20 @@ class SurveyInvite(models.TransientModel):
 
     applicant_id = fields.Many2one('hr.applicant', string='Applicant')
 
+    def _send_mail(self, answer):
+        mail = super()._send_mail(answer)
+        if 'applicant_id' in answer and answer.applicant_id:
```

Comment:
The field exists as it's defined just above :-D
```suggestion
        if answer.applicant_id:
```